src/dbnode/storage/namespace.go

```diff
@@ -158,6 +158,7 @@ type databaseNamespaceMetrics struct {
 	snapshot              instrument.MethodMetrics
 	write                 instrument.MethodMetrics
 	writeTagged           instrument.MethodMetrics
+	writeAggregated       instrument.MethodMetrics
```

Comment:
Think this didn't get renamed, this should be `aggregateTiles` yeah to match the new name?

Reply:
Renamed to `aggregateTiles`.